### DISCUSSION TITLE
how to delete very first commit

### DISCUSSION THREAD

**Discussion by mstfdkmn** [gitpython-developers/GitPython#1462] [Q&A] [answered]
I have not able to find an equivalent of `git update-ref -d HEAD` to remove the first commit made for an empty repository.

I guess the delete method might be used for this aim but could not manage to succeed it. https://github.com/gitpython-developers/GitPython/blob/main/git/refs/head.py#L139

Could you clarify this?
Thanks.

**Comment by Byron** [comment, marked as answer]
For special cases it should be easiest to run `repo.git.update_ref(d='HEAD')`. You can read more on how to use git directory [in the docs](https://gitpython.readthedocs.io/en/stable/tutorial.html#using-git-directly).

**Reply by mstfdkmn** [reply]
Thanks @Byron, I should have seen this.
